maintainer/format/cmake-format.sh

```diff
@@ -30,5 +30,3 @@ if ! ${CMAKE_FORMAT} --version | grep -qEo "${CMAKE_FORMAT_VER}"; then
     echo "Could not find cmake-format ${CMAKE_FORMAT_VER}."
     exit 2
 fi
-
-${CMAKE_FORMAT} "$@"
```

Comment:
I did not completely follow what happened on the SD side when the formatting tools were refactored by Michael and Kai. I you merged Michael's work, you'll need to revert that and merge Kai's work.

Reply:
Yes, I did merge with Kai's work. Unfortunately, I fixed the same issues on my own before that, so the merge was a big mess.